doc/cprover-manual/test-suite.md

```diff
@@ -213,6 +213,7 @@ The table below summarizes the coverage criteria that CBMC supports.
 Criterion |Definition
 ----------|----------
 assertion |For every assertion, generate a test that reaches it
+assume    |For every assume, generate a test to make sure that the state space before/after it hasn't been emptied
```

Comment:
Propose:
"For every assume, generate tests before and after the assume statement to indicate coverage before and after the assume statement"

We could also rewrite to be clearer about empty state space, but this is maybe not the clean mechanics, instead focusing on the motivation/theory.